rest_api/network_apis/route-route-openshift-io-v1.adoc

```diff
@@ -44,7 +44,6 @@ Required::
 
 | `metadata`
 | xref:../objects/index.adoc#objectmeta-meta-v1[`ObjectMeta meta/v1`]
-| 
```

Comment:
@xenolinux Did you intend to delete this line? The table doesn't look right in your preview.

Reply:
Addressed